tests/test_driver.py

```diff
@@ -185,37 +185,69 @@ def test_get_outputs(self, driver, carol_pubkey,
             'output_index': 0
         } in outputs
 
-    @mark.parametrize('spent,outputs_qty', ((False, 1), (True, 1), (None, 2)))
-    def test_get_outputs_with_spent_query_param(
-            self, spent, outputs_qty, driver, carol_pubkey,
-            persisted_carol_bicycle_transaction,
-            persisted_carol_car_transaction,
-            persisted_transfer_carol_car_to_dimi):
-        outputs = driver.outputs.get(carol_pubkey, spent=spent)
-        assert len(outputs) == outputs_qty
-
-        # Return only unspent outputs
-        if spent is False:
-            assert {
-                'transaction_id': persisted_carol_bicycle_transaction['id'],
-                'output_index': 0
-            } in outputs
-        # Return only spent outputs
-        elif spent is True:
-            assert {
-                'transaction_id': persisted_carol_car_transaction['id'],
-                'output_index': 0
-            } in outputs
-        # Return all outputs for carol
-        elif spent is None:
-            assert {
-                'transaction_id': persisted_carol_bicycle_transaction['id'],
-                'output_index': 0
-            } in outputs
-            assert {
-                'transaction_id': persisted_carol_car_transaction['id'],
-                'output_index': 0
-            } in outputs
+    def test_get_outputs_with_spent_query_param(self, driver):
+        from bigchaindb_driver.crypto import generate_keypair
+        import uuid
+
+        test_parameters = [(False, 2, 1), (True, 0, 1), (None, 2, 2)]
+
+        def create_transaction():
+            return driver.transactions.prepare(
+                    operation='CREATE',
+                    signers=carol.public_key,
+                    asset={
+                        'data': {
+                            'asset': {
+                                'serial_number': str(uuid.uuid4()),
+                                'manufacturer': str(uuid.uuid4()),
+                            },
+                        },
+                    },
+            )
+
+        carol, dimi = generate_keypair(), generate_keypair()
```

Comment:
It might sound dumb, but I would add the initial conditions of this test, that are:

```python
assert len(driver.outputs.get(carol.public_key, spent=True)) == 0
assert len(driver.outputs.get(carol.public_key, spent=False)) == 0
assert len(driver.outputs.get(carol.public_key, spent=None)) == 0
```

Reply:
I'll change it but do you have a specific reason?

Reply:
In this test we check that the `outputs` endpoint works as expected. To me, another condition to check is that there are no outputs for a freshly generated keypair, something like "when I start, the outputs are zero, then I add an asset and I expect 1, then ..."

Reply:
Yes, that makes sense!